Common/Data/Convert/SmallDataConvert.h

```diff
@@ -82,6 +82,20 @@ inline void Uint8x3ToFloat4(float f[4], uint32_t u) {
 #endif
 }
 
+inline void Uint8x3ToFloat3(float f[4], uint32_t u) {
+#if defined(_M_SSE) || PPSSPP_ARCH(ARM_NEON)
+	float temp[3];
```

Comment:
Is this safe? Shouldn't it be `float temp[4]`?

Reply:
good catch, thanks!